requirements.txt

```diff
@@ -1,17 +1,18 @@
-BeautifulSoup4==4.9.0
-dispatcher==1.0
-dispatch==0.7.7
-pexpect==4.8.0
-psutil==5.6.6
-pycrypto==2.6.1
-pyserial==3.4
-usbinfo==1.1.1
-pyudev==0.21.0
-netifaces==0.10.7
-numpy==1.15.2
-PyYAML==5.3.1
-PyDispatcher==2.0.5
-dpkt==1.9.1
-jsonfield==2.0.2
-urllib3==1.24.2
-ipaddress==1.0.22
+BeautifulSoup4 == 4.9.0
+dispatch == 0.7.7
```

Comment:
If we install in following order then some files of dispatch package are overwritten by dispatcher. 
dispatch==0.7.7
dispatcher==1.0

Thus the ordering of these 2 is intentional as below:
dispatcher==1.0
dispatch==0.7.7

Similarly for following, usbinfo 1.1.1 installs pyudev 0.22.0 which has a bug. Due to which usbinfo is not working. Thus following ordering is also intentional.
 usbinfo==1.1.1
pyudev==0.21.0

Reply:
Ok, seems like a fragile behavior. I'll add a note to the file.